### DISCUSSION TITLE
Is it possible to disable the INFO line that is printed when a player joins the server?

### DISCUSSION THREAD

**Discussion by Technerder** [Minestom/Minestom#1030] [Help]
Referring to: 
`> [ForkJoinPool.commonPool-worker-1] [14:54:47] (EncryptionResponsePacket.lambda$process$0) - INFO - UUID of player Username is UUID`

Note: ideally I'd like a solution that preserves the `Starting Minestom server.` and `Minestom server started successfully.` lines

**Comment by dev-hydrogen** [comment]
This would be easiest by cloning or forking the repo and removing this line: https://github.com/Minestom/Minestom/blob/master/src/main/java/net/minestom/server/network/packet/client/login/EncryptionResponsePacket.java#L81
This would be way more effort than its worth, since you'd have to update the fork/clone manually.
UUID's or usernames aren't very "secret" and I don't see a lot of point to this other than removing a single line in logs when a player joins.
